driver/src/test/java/org/example/driver/unit/controller/DriverControllerTest.java

```diff
@@ -175,7 +175,8 @@ void create_whenValidInput_thenMapsToBusinessModel() throws Exception {
             ArgumentCaptor<DriverCreateEditDto> driverCaptor = ArgumentCaptor.forClass(DriverCreateEditDto.class);
 
             verify(driverService, times(1)).create(driverCaptor.capture());
-            assertThat(driverCaptor.getValue().name()).isEqualTo(DEFAUlT_NAME);
+            assertThat(driverCaptor.getValue().firstName()).isEqualTo(DEFAUlT_NAME);
+            assertThat(driverCaptor.getValue().lastName()).isEqualTo(DEFAUlT_NAME);
```

Comment:
Здесь и далее - в имя константы попал символ нижнего регистра:
```suggestion
            assertThat(driverCaptor.getValue().firstName()).isEqualTo(DEFAULT_NAME);
            assertThat(driverCaptor.getValue().lastName()).isEqualTo(DEFAULT_NAME);
```